front/src/data/LogsData.ts

```diff
@@ -0,0 +1,45 @@
+import { llm_logs } from "@prisma/client";
+import { cache } from "react";
+
+export const LogsData = {
+    getLogs: cache(async (): Promise<llm_logs[]> => {
```

Comment:
I think we need to be careful with cache, when we start fetching real data

Reply:
yes, on second thought I don't think that is necessary - was following [this example](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-server-with-third-party-libraries) because I thought cache was required to make it server-side, but it is not 

will push a new commit, good catch!